README.org

```diff
@@ -5,12 +5,12 @@ Emacs major mode for editing [[https://xon.sh][xonshrc]] files. This mode inheri
 
 * Installation
 
-For now just copy the ~xonsh-mode.el~ file to your ~~/.emacs.d/elisp/~
+For now just copy the xonsh-mode.el file to your ~/.emacs.d/elisp/
 folder and then make sure the following line is executed on startup:
 
 #+begin_src elisp
 (add-to-list 'load-path "~/.emacs.d/elisp/")
 #+end_src
 
 Which if you are using the scheme above would be in the
-~~/.emacs.d/init.el~ folder.
```

Comment:
My only real question is why remove the org-markup here?

Reply:
Because github wasn't rendering it and that is probably the only place people will read it from https://github.com/salotz/xonsh-mode/tree/9cc48a401b7bbfb4303ec44d9192be75f6c33ba7